.github/workflows/pars-upload-branch.yaml

```diff
@@ -10,7 +10,9 @@ on:
 
 env:
   NEXT_PUBLIC_DISABLE_AUTH: "true"
-  PARS_UPLOAD_URL: "http://localhost:8000"
+  PARS_UPLOAD_URL: "http://localhost:8000/pars"
+  # above is for cypress, below is for next.js
```

Comment:
if we're gonna separate, we could use the `CYPRESS_*` pattern to save passing them in, but doesn't have to block this PR